criteria/document-maturity.md

```diff
@@ -11,13 +11,15 @@ redirect_from:
 ## Requirements
 
 * A codebase MUST be versioned.
-* A codebase that is ready to use MUST only depend on other codebases that are also ready to use.
-* A codebase that is not yet ready to use MUST have one of the labels: prototype, alpha, beta or pre-release version.
+* A codebase version that is ready to use MUST only depend on other codebase versions that are also ready to use.
+* A codebase which has no versions ready for use MUST document that clearly.
+* A codebase version that is not yet ready to use MUST use semantic versioning with a 0 for the major version number or have one of the labels: pre-alpha, alpha, beta or release candidate.
```

Comment:
I am not sure I understand the added value of mandating specific labels, unless they match specific phases that are mandated by the Standard, in which case a link would seem appropriate to me. Some projects like to use `rc` (for “release candidate”), some others “golden build”, “canary”, “discovery”… On top, some package managers will not allow spaces in labels (for example, tags in NPM, IIRC), and `release candidate` as currently written seems to be a spec item.

My suggestion for solving most comments I leave here would be to stick to SemVer.

Reply:
I would strongly encourage most codebases to stick to SemVer, but there are codebases for which this does not make sense. One examples would be a codebase for a website which automatically deploys from `develop` to `live` if the tests are green, another would be a rolling update linux distribution. These are, however, the exceptions. SemVer is well suited to most codebases.

Reply:
> I am not sure I understand the added value of mandating specific labels, unless they match specific phases that are mandated by the Standard, in which case a link would seem appropriate to me.

I think this part was an insightful observation. Let's think a bit more about what is really adding value to those who come looking at a codebase.

Reply:
The current version of this PR no longer mandates specific labels.